src/cgpt/cgpt_common.c

```diff
@@ -615,8 +615,8 @@ const Guid guid_mswin_data =        GPT_ENT_TYPE_MSWIN_DATA;
 
 static struct {
   const Guid *type;
-  char *name;
-  char *description;
+  char name[GUID_STRLEN];
+  char description[GUID_STRLEN];
```

Comment:
This will produce a non-null-terminated string if the initializer is exactly `GUID_STRLEN` characters.

Reply:
I don't see a great way to fix this at initialization time, so `strncpy`/`strncmp` might be a better approach.